src/main/java/Wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package Wolox.training.repositories;
+
+import Wolox.training.models.Book;
+import org.springframework.data.repository.Repository;
+import java.util.Optional;
+
+public interface BookRepository extends Repository<Book, Long> {
+
+    Optional<Book> findByAuthor(String author);
```

Comment:
Spring has a set of interfaces per repository domain that already declare CRUD methods, for example `CrudRepository` and `JpaRepository`. With `JpaRepository` it extends from `CrudRepository` and also the paging and sorting functionality, so you don't need to declare the `findById` function.
```suggestion
public interface BookRepository extends JpaRepository<Book, Long> {
```

Reply:
I checked that this change was made in the PR #4